src/ros_filter.cpp

```diff
@@ -1887,6 +1892,14 @@ namespace RobotLocalization
   }
 
   template<typename T>
+  bool RosFilter<T>::disableSrvCallback(std_srvs::Empty::Request& request,
+                            std_srvs::Empty::Response&)
```

Comment:
I was just following the style in the rest of the file. But I already changed it so all good.